<launch>
  <node pkg="hik_ros1_camera_driver" type="hik_ros1_camera_driver_node" name="hik_ros1_camera_driver" output="screen">
    <param name="camera_type" type="str" value="GigE" />
    <!-- <param name="camera_type" type="str" value="USB" /> -->
    <!-- <param name="camera_serial" type="str" value="G57500000" /> -->
    <param name="camera_frame_id" type="str" value="camera_link" />
    <param name="camera_info_path" type="str" value="package://hik_ros1_camera_driver/cfg/camera_info.yaml" />
  </node>
</launch>